Themis/ViewModels/Assessment/Programming/CodeEditorViewModel.swift

```diff
@@ -291,7 +285,7 @@ class CodeEditorViewModel: ExerciseRendererViewModel {
                 let endLine = lines[1]
                 if fileLines.count >= endLine {
                     let startIndex = fileLines[startLine - 1].location
-                    let endIndex = fileLines[endLine - 1].location + fileLines[endLine - 1].length
+                    let endIndex = fileLines[endLine - 1].location + fileLines[endLine - 1].length - 1
```

Comment:
What's the difference between this line and line 278? Why do we need to subtract 1 from this line here?

Reply:
There's no difference. I added it to line 278 and refactored it a bit to make it easier to understand. 

The reason for subtracting 1 is that the `CodeEditor` cannot show a highlight when the last index of the file is included in the highlight. I'm not sure about the exact reason why it happens, though.